scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -823,7 +826,8 @@ class ScalametaParser(input: Input, dialect: Dialect) { parser =>
       token.is[KwThis] || token.is[KwThrow] || token.is[KwTry] || token.is[KwWhile] ||
       token.is[LeftParen] || token.is[LeftBrace] || token.is[Underscore] ||
       token.is[Unquote] || token.is[MacroSplice] || token.is[MacroQuote] ||
-      token.is[MacroQuotedIdent] || token.is[MacroSplicedIdent]
+      token.is[MacroQuotedIdent] || token
+        .is[MacroSplicedIdent] || token.is[Indentation.Indent]
```

Comment:
reformat to `token.is[MacroSplicedIdent] || <LF> token.is[Indentation.Indent]`

Reply:
Fixed